src/react-components/src/components/03-organisms/Hero/Hero.stories.tsx

```diff
@@ -0,0 +1,90 @@
+import * as React from "react";
+
+import Image from "../../01-atoms/Images/Image/Image";
+import Heading from "../../01-atoms/Text/Headings/Heading";
+import { HeroTypes } from "./HeroTypes";
+import Hero from "./Hero";
+import SectionName from "../../00-base/SectionName";
+import Placeholder from "../../00-base/Placeholder";
+
+export default {
+  title: "Hero",
+  component: Hero,
+};
+
+export const heroPrimary = () => <Hero
+  heroType={HeroTypes.Primary}
```

Comment:
(may be out of scope) I'd love to figure out a way to make it clear that users are supposed to be only using this enum, and how to do that ... Does Typescript throw a useful error when passing in a string or does it let it through?

Reply:
Passing in a string should throw an error since the `heroType` prop is defined as `HeroTypes`. I don't know of a clear way to let users know that it's supposed to be an enum, but maybe reading the component itself is enough?

Reply:
OK!  In that case all we need is to be exporting `HeroType` in `index.ts`.

Reply:
Exported `HeroTypes` in `index.ts`.